examples/graph-demo/index.html

```diff
@@ -201,6 +88,297 @@
+              "shadow": {
+                "type": "math_number",
+                "fields": {
+                  "NUM": 10
+                }
+              }
+            }
+          }
+        },
+        {
+          "kind": "block",
+          "type": "math_constrain",
+          "inputs": {
+            "VALUE": {
+              "shadow": {
+                "type": "math_number",
+                "fields": {
+                  "NUM": 50
+                }
+              }
+            },
+            "LOW": {
+              "shadow": {
+                "type": "math_number",
+                "fields": {
+                  "NUM": 1
+                }
+              }
+            },
+            "HIGH": {
+              "shadow": {
+                "type": "math_number",
+                "fields": {
+                  "NUM": 100
+                }
+              }
+            }
+          }
+        },
+        {
+          "kind": "block",
+          "type": "math_random_int",
+          "inputs": {
+            "FROM": {
+              "shadow": {
+                "type": "math_number",
+                "fields": {
+                  "NUM": 1
+                }
+              }
+            },
+            "TO": {
+              "shadow": {
+                "type": "math_number",
+                "fields": {
+                  "NUM": 100
+                }
+              }
+            }
+          }
+        },
+        {
+          "kind": "block",
+          "type": "math_random_float"
+        },
+        {
+          "kind": "block",
+          "type": "math_atan2",
+          "inputs": {
+            "X": {
+              "shadow": {
+                "type": "math_number",
+                "fields": {
+                  "NUM": 1
+                }
+              }
+            },
+            "Y": {
+              "shadow": {
+                "type": "math_number",
+                "fields": {
+                  "NUM": 1
+                }
+              }
+            }
+          }
+        },
+      ]
+    },
+    {
+      "kind": "category",
+      "name": "Variables",
+      "colour": "%{BKY_VARIABLES_HUE}",
+      "contents": [
+        {
+          "kind": "block",
+          "type": "graph_get_x",
+        }
+      ]
+    },
+    {
+      "kind": "category",
+      "name": "Logic",
+      "colour": "%{BKY_LOGIC_HUE}",
+      "contents": [
+        {
+          "kind": "block",
+          "type": "logic_compare"
+        },
+        {
+          "kind": "block",
+          "type": "logic_operation"
+        },
+
+        {
+          "kind": "block",
+          "type": "logic_negate"
+        },
+        {
+          "kind": "block",
+          "type": "logic_boolean"
+        },
+        {
+          "kind": "block",
+          "type": "logic_ternary"
+        }
+      ]
+    }
+  ]
+};
+
+const startBlocks = {
+  "blocks": {
+    "blocks": [{
+      "kind": "block",
+      "type": "graph_set_y",
+      "deletable": false,
```

Comment:
I think this can be removed since it doesn't apply in json

Reply:
Ah, darn, I thought I removed that.